course/layouts/partials/course_content.html

```diff
@@ -1,14 +1,25 @@
+{{ $data := .context }}
+
+{{/*  scale images only for instructor insights.   */}}
+{{ if isset . "isInstructorInsightsPage" }}
+  <style>
+    img {
+      width: 100%;
```

Comment:
This inline style is not looking much good to me, please let me know if anyone has a more suitable solution.